opentelemetry-exporter-gcp-trace/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -357,24 +364,32 @@ def _strip_characters(ot_version):
 }
 
 
-def _extract_resources(resource: Resource) -> Dict[str, str]:
+def _extract_resources(
+    resource: Resource, otel_env_resource: Optional[Resource] = None
+) -> Dict[str, str]:
+    extracted_attributes = {}
+    if otel_env_resource:
+        extracted_attributes.update(otel_env_resource.attributes)
     resource_attributes = resource.attributes
     if resource_attributes.get("cloud.provider") != "gcp":
-        return {}
+        return extracted_attributes
     resource_type = resource_attributes["gcp.resource_type"]
     if (
         not isinstance(resource_type, str)
         or resource_type not in OT_RESOURCE_ATTRIBUTE_TO_GCP
     ):
-        return {}
-    return {
-        "g.co/r/{}/{}".format(resource_type, gcp_resource_key,): str(
-            resource_attributes[ot_resource_key]
-        )
-        for ot_resource_key, gcp_resource_key in OT_RESOURCE_ATTRIBUTE_TO_GCP[
-            resource_type
-        ].items()
-    }
+        return extracted_attributes
+    extracted_attributes.update(
+        {
+            "g.co/r/{}/{}".format(resource_type, gcp_resource_key,): str(
```

Comment:
Only certain resource attributes are recognized by Google with this `g.co/r/` format. The ones that pass the regex can just be added straight in without changing the keys.

Reply:
This was extracted [directly from `main`](https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/blob/master/opentelemetry-exporter-gcp-trace/src/opentelemetry/exporter/cloud_trace/__init__.py#L370).  I can change this, but what format at you looking for exactly.

Reply:
The `g.co/r/` attributes should only correspond to monitored resource labels (e.g. [gce_instance](https://cloud.google.com/monitoring/api/resources#tag_gce_instance)). The additional resource labels we are putting can just be copied over from the OTel resource in the span.

Reply:
@aabmass let me know if this is what you want now.  For `gce_instance` I kept it the same as before but for `gke_container` I only use the `ot_resource_key` (for example `k8s.cluster.name`).